spec/configuration_spec.rb

```diff
@@ -47,70 +47,70 @@
       end
     end
 
-    describe "#minimum_coverage" do
+    shared_examples "checks coverage settings" do |coverage_setting|
```

Comment:
I feel like "setting coverage expections" or something would be a better name (we also check the results not just that what is given is correct)